app/views/comment/index.html.haml

```diff
@@ -0,0 +1,3 @@
+= turbo_frame_tag "ab-comments-list-#{a.id}" do
+  %div{ id: "ab-smiles-#{a.id}" }= render "answerbox/smiles", a:
```

Comment:
Please don't render the smiles inside the comments frame, they just don't belong here.

Render them in the card footer for now and we'll move them to their own frame later.

Reply:
13775c6